src/errors.rs

```diff
@@ -114,6 +117,14 @@ pub(crate) enum RustupError {
         component: String,
         suggestion: Option<String>,
     },
+    #[error("toolchain '{}' does not support target '{}'{}", .desc, .target,
+    "\nnote: you can see a list of supported targets with `rustc --print=target-list`\n\
+    note: if you are adding support for a new target to rustc itself, see https://rustc-dev-guide.rust-lang.org/building/new-target.html
+    ".to_string())]
```

Comment:
Using {} to add in a constant string feels weird. Any reason we can't just put it in the main format string? (perhaps using a multi-line string for easier reading)?

Reply:
Fixed. Thanks! Make sense.